pages/checkout/pay/paypal.vue

```diff
@@ -25,11 +25,6 @@ useHead({
 });
 
 useAutoLogout();
-
-onBeforeRouteLeave((to, _from, next) => {
-  next(false);
-  window.location.href = to.fullPath;
-});
```

Comment:
Any reason this was removed?

Reply:
For paypal I am not so sure but for card payments in general we should make sure that navigating to and from `/card` does a request to server when navigating.

This is because we want to have a valid CSP(Content Security Policy) for the site, but unfortunately for 3DSecure 2 we have to relax it because each bank is using their own domains.

Reply:
I didn't encounter any issues not doing full reload, but if it is required for CSP I could return it.

Reply:
So, going to card payment triggers a hard reload using the middleware that will load the relaxed CSP, then the whole site will work with the relaxed CSP unless you do a hard reload.

So you won't encounter any problems, but I would rather not have relaxed CSP rules outside the card payment flow.

To test this make sure you don't have the SKIP_CSP env variable and then check if in card payment it uses a more relax ruleset than the rest of the website.

Reply:
reverted.